### DISCUSSION TITLE
Support for floating-point values

### DISCUSSION THREAD

**Discussion by MagmarFire** [RetroAchievements/RAIntegration#793] [Completed]
There are a few games that utilize floating-point values for certain things, like timers or other things that may represent fractional numbers that, for one reason or another, the original game devs didn't elect to represent with integer values. This becomes especially more common with later-generation game consoles, and as more and more of those get achievement support, it would be a huge benefit to achievement developers to be able to act on memory addresses as if they were floating-point values.

**Comment by Jamiras** [comment]
#827
